sdk/samples/iot/hub/windows/how_to_iot_hub_samples_windows.md

```diff
@@ -72,24 +69,21 @@ _The following was run on Microsoft Windows 10.0.18363.836._
 05. Clone the Azure Embedded SDK for C
 
     ```shell
+    C:\>cd..
```

Comment:
`cd`'ing from the root backwards won't do much. I think you meant for this to be a continuation from the previous step with `vcpkg`?
The next line shows `C:\>` which is at the root which I know is a subtle detail. It does show where the user should be though.